0107-preventing-gaming-within-the-mobile-network.md

```diff
@@ -0,0 +1,81 @@
+
+
+## Motivation
+Currently, the first MOBILE Service Provider and company that reviews CPI information, Nova Labs (d.b.a. Helium Mobile), has expressed concern for ongoing gaming that they've identified within the MOBILE subnetwork; however, they do not have the appropriate authority to stop this from occurring. This HIP will give current and future MOBILE Service Providers the authority needed to curb gaming.
+
+## Stakeholders
+The stakeholders of this proposal are:
+
+- **Service Providers** will be able to temporarily turn off CBRS radios/Wi-Fi APs suspected of gaming. This may require them to allocate internal resources to use these tools to manage the rejection communication.
+- **CPI Reviewing Entity** will be able to reject previously approved CPI submissions if they are suspected of gaming.
+- **All Deployers** will now suffer an opportunity cost loss when they are caught gaming.
+- **Outdoor Wi-Fi Deployers** will now be required to upload a picture of their deployment when they assert a new location
+
+
+## Detailed Explanation
+This HIP proposes allowing Service Providers (Service Provider is defined as an entity who was approved by the MOBILE network via HIP and has staked 500M MOBILE tokens) the ability to temporarily deactivate CBRS Radios and Wi-Fi access points when they have a reasonable belief that gaming is occurring. Further, the Certified Professional Installer (CPI) Entity may revoke or deny submissions due to suspected gaming.
+
+Specifically, Service Providers may stop radios/access points from producing heartbeats if they are suspected of gaming, effectively setting their reward multiplier to `0X` for both PoC and Data Transfer rewards. This may be set on CBRS Radios and Wi-Fi access points for up to ten (`10`) consecutive epochs. The specific reasons are identified in the section below. The reason and length of deactivation must be publicly submitted to the MOBILE Oracle. After the defined deactivation time (10 epochs maximum), the device must be automatically returned to normal participation on the network (with the exception of outdoor CBRS radios, which need a newly approved CPI submission). However, if the gaming activity continues, the Service Provider may deactivate the device again using the same duration defined above.
+
+Outdoor Radios suspected of gaming may have their CPI registration revoked or denied, and will not be able to earn rewards until a new submission is submitted and approved.
+
+### Outdoor Wi-Fi Maximum Height
+It is worth noting that due to the shorter range of outdoor Wi-Fi, the maximum asserted height will be changed to 100 feet. This will be enforced initially in MOBILE oracles by ignoring values over 100 feet and may, in the future be blocked at the protocol level. Deployments can still be deployed higher than 100 feet but these values will not be considered for the purpose of modeled coverage rewards.
```

Comment:
@madninja how is height currently submitted for wifi hotspots? There's no concept of elevation on-chain for mobile, only IOT.

Reply:
We should scope adding it to outdoor WiFi Hotspots.